File: namsor/php/src/SubclassificationIndianFullExample.php
<?php

namespace OSEG\NamsorExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use Namsor;

$config = Namsor\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("X-API-KEY", "YOUR_API_KEY");

try {
    $response = (new Namsor\Client\Api\IndianApi(config: $config))->subclassificationIndianFull(
        full_name: "Jannat Rahmani",
    );

    print_r($response);
} catch (Namsor\Client\ApiException $e) {
    echo "Exception when calling IndianApi#subclassificationIndianFull: {$e->getMessage()}";
}

File: namsor/ruby/src/SubclassificationIndianFullExample.rb
require "json"
require "namsor_client"

NamsorClient.configure do |config|
    config.api_key["X-API-KEY"] = "YOUR_API_KEY"
end

begin
    response = NamsorClient::IndianApi.new.subclassification_indian_full(
        "Jannat Rahmani", # full_name
    )

    p response
rescue NamsorClient::ApiError => e
    puts "Exception when calling IndianApi#subclassification_indian_full: #{e}"
end
